### ISSUE TITLE
Rotate screen - Menu is off center and cannot scroll

### ISSUE THREAD

**Issue by srepussard** [cds-snc/covid-shield-mobile#212]
Environments :
Android v6 on Galaxy S7 (BrowserStack, apk)

Scenario : 
- Open app on phone
- Open Covid Alert menu
- Rotate phone to horizontal position

Issue : 
Menu is off-center on the screen and we cannot scroll to the bottom of the menu so cannot access the rest of the functionalities

![bs_realdroid_Mobile_Samsung Galaxy S7-6 0-1440x2560 (1)](https://user-images.githubusercontent.com/47008295/85894484-b0b17f80-b7a9-11ea-9b4b-91e51d726741.jpg)

**Comment by mmiron-HC** [comment]
iPhone XR has the same issue.

**Comment by JuliannaR** [comment]
Currently tracking for release 1 orientation to be locked. We are working on a solution but there is some work happening upstream to address. @srepussard can you confirm I can close this issue for now?

**Comment by srepussard** [comment]
I confirm that the app has the rotation functionality off : turning the phone doesn't trigger the screen's rotation anymore
